apex/transformer/parallel_state.py

```diff
@@ -126,13 +175,13 @@ def initialize_model_parallel(
     with a total of 16 GPUs, rank 0 to 7 belong to the first box and
     ranks 8 to 15 belong to the second box.
     """
+    default_nccl_net = os.getenv("NCCL_NET")
```

Comment:
Q: is this variable used other than the epilogue of this function?

Reply:
Not, it should not be used anywhere else than in epilogue

Reply:
then could you move this definition to right before the epilogue for a little bit better clarity?

Reply:
I can move it down a little bit, but no further than the first `new_process_group` where `NCCL_NET` is going to be modified.